metadata.rb

```diff
@@ -5,7 +5,7 @@
 maintainer "John Alberts"
 maintainer_email "john@alberts.me"
 description "Mounts attached EBS volumes"
-version "0.3.12"
+version "0.3.13"
```

Comment:
why not `0.3.12-evg1`?

Reply:
The public version is 0.3.6.  We're 6 minor versions ahead of them.  I'm not sure why we weren't "-evg1"ing the previous versions, but I figured it doesn't matter too much

Reply:
I like the `evg` convention because we can immediately tell if something is a fork, but since we have already been ignoring the convention for this lib for the past 8 years, I agree it doesn't matter much.  :-)